index.bs

```diff
@@ -4071,13 +4070,11 @@ even if the SafetyNet API is also present.
     as follows:
     - Verify that |attStmt| is valid CBOR conforming to the syntax defined above and perform CBOR decoding on it to extract the
         contained fields.
-    - Verify that |response| is a valid SafetyNet response of version |ver|.
+    - Verify that |response| is a valid SafetyNet response of version |ver| 
+        by following steps indidated by the [SafetyNet online documentation](https://developer.android.com/training/safetynet/attestation.html#compat-check-response).
```

Comment:
s/following steps indidated/following the steps indicated/

Reply:
done